src/applications/appeals/995/utils/submit.js

```diff
@@ -49,7 +49,7 @@ export const getClaimantData = ({
   };
 
   if (result.claimantType === 'other' && claimantTypeOtherValue) {
-    result.claimantTypeOtherValue = (claimantTypeOtherValue || '').substring(
+    result.claimantTypeOtherValue = claimantTypeOtherValue.substring(
```

Comment:
Was this failing before?

Reply:
I removed the empty string fallback because there isn't any way to test it because the check in the `if` statement in the line above checks if `getClaimantData` is a falsy value. So, there's no way to add code coverage for the empty string fallback.